lib/collection/src/operations/conversions.rs

```diff
@@ -796,6 +796,8 @@ impl From<QueryEnum> for api::grpc::qdrant::QueryEnum {
                     },
                 )),
             },
+            QueryEnum::Discover(_) => todo!("luis in other PR"),
+            QueryEnum::Context(_) => todo!("luis in other PR"),
```

Comment:
Do we guarantee not to call this in the current implementation?

Reply:
Yes, we haven't exposed any apis at this point, so there would be no reason that this variant would be run